### DISCUSSION TITLE
zscore-normalize: error: unrecognized arguments for output

### DISCUSSION THREAD

**Discussion by Jkgarciam** [jcreinhold/intensity-normalization#72] [Q&A] [answered]
HI I keep getting this " zscore-normalize: error: unrecognized arguments:" from my output value, also this is happening with my mac laptop, I've tried doing it in a windows laptop and it works fine, any ideas what could be happening? Thanks.
P.S: Im doing it from the CLI

**Comment by jcreinhold** [comment, marked as answer]
I can't help unless you provide more information. Please copy the below form to a comment and fill out all of the sections:

```
<!-- A clear, concise description of what the bug is. -->

## To Reproduce

Steps to reproduce the behavior:

1.
2.
3.

<!-- Provide code sample, error messages, and stack traces here. -->

## Expected behavior

<!-- A clear, concise description of what you expected to happen. -->

## Environment

 - intensity-normalization version (e.g., 0.3.8):
 - numpy version (e.g., 1.0):
 - OS (e.g., Linux):
 - How you installed intensity-normalization (`conda`, `pip`, source):
 - Build command you used (if installed from source):
 - Python version:
 - Any other relevant information:

## Additional context

<!-- Add any other context about the problem here. -->
```

**Comment by Jkgarciam** [comment]
Hi, I just resolved, my path was wrong ( I had one folder as output but it was inside of another that I'd missed. 
Thank you anyway!
